stopify-continuations/src/callcc/callcc.ts

```diff
@@ -73,15 +79,21 @@ const visitor: Visitor = {
       h.transformFromAst(path, [anf]));
     timeSlow('declVars', () =>
       h.transformFromAst(path, [declVars]));
-    timeSlow('delimit', () =>
-      h.transformFromAst(path, [[delimitTopLevel, opts]]));
+    // If stopifying eval'd string at runtime, don't delimit statements so that
+    // we can suspend through the eval.
+    if (!(<any>opts).renames) {
+      timeSlow('delimit', () =>
+        h.transformFromAst(path, [[delimitTopLevel, opts]]));
+    }
     timeSlow('label', () =>
       h.transformFromAst(path, [label.plugin]));
     timeSlow('jumper', () =>
       h.transformFromAst(path, [[jumper.plugin, opts]]));
 
+    path.stop();
```

Comment:
@baxtersa I want to confirm that this is just an optimization.

Reply:
yes